addons/web/static/src/js/views/abstract_view.js

```diff
@@ -133,20 +133,19 @@ var AbstractView = Class.extend({
     getController: function (parent) {
```

Comment:
[FIX] web: Views: always destroy BasicModel instances

Before this rev., it might happen that the parent of BasicModel instances
wasn't correctly set to the Controller instance. It was actually set to the
ActionManager instance, which is never destroyed. So when this happened,
the BasicModel instance wasn't destroyed when it's controller was.